pkg/syncer/spec/spec_process.go

```diff
@@ -333,6 +333,7 @@ func (c *Controller) applyToDownstream(ctx context.Context, gvr schema.GroupVers
 	// Strip cluster name annotation
 	downstreamAnnotations := downstreamObj.GetAnnotations()
 	delete(downstreamAnnotations, logicalcluster.AnnotationKey)
+	delete(downstreamAnnotations, workloadv1alpha1.InternalClusterStatusAnnotationPrefix+c.syncTargetKey)
```

Comment:
Could you add a TODO that it could be removed when switching to Syncer Virtual Workspace transformations ?

Reply:
done!